multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -90,21 +90,17 @@ write_files:
     owner: root:root
     content: |
       #!/bin/bash -e
-      /usr/bin/curl -H "Content-Type: application/json" -XPOST -d @"/srv/kubernetes/manifests/kube-system.json" "http://127.0.0.1:8080/api/v1/namespaces"
+      docker run -v /etc/kubernetes:/etc/kubernetes \
+                 -v /srv/kubernetes:/srv/kubernetes \
+                 --net host \
+                 --rm lachlanevenson/k8s-kubectl \
+                 apply -f /srv/kubernetes/manifests/kube-system.json
 
-      /usr/bin/curl  -H "Content-Type: application/json" -XPOST \
-      -d @"/srv/kubernetes/manifests/kube-dns-rc.json" \
-      "http://127.0.0.1:8080/api/v1/namespaces/kube-system/replicationcontrollers"
-
-      /usr/bin/curl  -H "Content-Type: application/json" -XPOST \
-      -d @"/srv/kubernetes/manifests/heapster-dc.json" \
-      "http://127.0.0.1:8080/apis/extensions/v1beta1/namespaces/kube-system/deployments"
-
-      for manifest in {kube-dns,heapster}-svc.json;do
-          /usr/bin/curl  -H "Content-Type: application/json" -XPOST \
-          -d @"/srv/kubernetes/manifests/$manifest" \
-          "http://127.0.0.1:8080/api/v1/namespaces/kube-system/services"
-      done
+      docker run -v /etc/kubernetes:/etc/kubernetes \
+                 -v /srv/kubernetes:/srv/kubernetes \
+                 --net host \
+                 --rm lachlanevenson/k8s-kubectl \
+                 apply -f /srv/kubernetes/manifests/
```

Comment:
I don't think we can safely use apply, as this is run during every boot. The api objects could have changed during their lifecycle and it's not safe for us to assume we can stomp on those changes.